src/Lexer.js

```diff
@@ -135,6 +134,15 @@ export class Lexer {
     if (this.options.pedantic) {
       src = src.replace(/^ +$/gm, '');
     }
+
+    if (this.options.pedantic) {
```

Comment:
Can you combine this if statement with the one above? I think we should do the tab conversion before replacing a line with only spaces with a blank line.